src/tests/functional/shared_test_classes/src/single_op/stft.cpp

```diff
@@ -0,0 +1,91 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/stft.hpp"
+
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include "shared_test_classes/base/utils/ranges.hpp"
+
+namespace ov {
+namespace test {
+
+std::string STFTLayerTest::getTestCaseName(const testing::TestParamInfo<STFTParams>& obj) {
+    std::ostringstream result;
+    using namespace std;
+    std::vector<InputShape> data_shapes;
+    int64_t frame_size;          // frame size value
+    int64_t frame_step;          // frame step value
+    bool transpose_frames;       // transpose_frames
+    ElementType data_type;       // data type
+    ElementType step_size_type;  // size/step type
+    utils::InputLayerType param_type;
+    ov::test::TargetDevice dev;
+    std::tie(data_shapes, frame_size, frame_step, transpose_frames, data_type, step_size_type, param_type, dev) =
```

Comment:
Please use references instead via std::get<n>(params) to avoid copying of parameters.

Reply:
Done, updated to use get and references where relevant (not for primitive types).